docs/user-guide/petset/bootstrapping/index.md

```diff
@@ -139,7 +141,8 @@ The role of the peer finder:
 You can invoke the peer finder inside the Pets we created in the last example:
 
 ```shell
-$ kubectl exec -it vm-0 /bin/bash
```

Comment:
I thought for consistency sake, we'd stick with `sh`. Is there a particular reason why `bash` was chosen for this example? If that's the case, I'll let it be the way it is.